back/config/initializers/reload_cl_settings.rb

```diff
@@ -0,0 +1,34 @@
+# Reloads the CL_SETTING_* environment variables
+# into the current seed data.
+
+if !CitizenLab.ee?
+  config = AppConfiguration.instance
+  settings = config.settings
+
+  config.host = ENV.fetch('CL_SETTINGS_HOST') if ENV.fetch('CL_SETTINGS_HOST', false)
+  
+  settings['core']['timezone'] = ENV.fetch('CL_SETTINGS_CORE_TIMEZONE') if ENV.fetch('CL_SETTINGS_CORE_TIMEZONE', false)
+  settings['core']['currency'] = ENV.fetch('CL_SETTINGS_CORE_CURRENCY') if ENV.fetch('CL_SETTINGS_CORE_CURRENCY', false)
+  settings['core']['reply_to_email'] = ENV.fetch('DEFAULT_FROM_EMAIL') if ENV.fetch('DEFAULT_FROM_EMAIL', false)
+
+  settings['facebook_login']['enabled'] = (ENV.fetch('CL_SETTINGS_FACEBOOK_LOGIN_ENABLED') == 'true') if ENV.fetch('CL_SETTINGS_FACEBOOK_LOGIN_ENABLED', false)
+  settings['facebook_login']['app_id'] = ENV.fetch('CL_SETTINGS_FACEBOOK_LOGIN_APP_ID') if ENV.fetch('CL_SETTINGS_FACEBOOK_LOGIN_APP_ID', false)
+  settings['facebook_login']['app_secret'] = ENV.fetch('CL_SETTINGS_FACEBOOK_LOGIN_APP_SECRET') if ENV.fetch('CL_SETTINGS_FACEBOOK_LOGIN_APP_SECRET', false)
+
+  settings['google_login']['enabled'] = (ENV.fetch('CL_SETTINGS_GOOGLE_LOGIN_ENABLED') == 'true') if ENV.fetch('CL_SETTINGS_GOOGLE_LOGIN_ENABLED', false)
+  settings['google_login']['client_id'] = ENV.fetch('CL_SETTINGS_GOOGLE_LOGIN_CLIENT_ID') if ENV.fetch('CL_SETTINGS_GOOGLE_LOGIN_CLIENT_ID', false)
+  settings['google_login']['client_secret'] = ENV.fetch('CL_SETTINGS_GOOGLE_LOGIN_CLIENT_SECRET') if ENV.fetch('CL_SETTINGS_GOOGLE_LOGIN_CLIENT_SECRET', false)
+
+  settings['maps']['map_center']['lat'] = ENV.fetch('CL_SETTINGS_MAPS_MAP_CENTER_LAT') if ENV.fetch('CL_SETTINGS_MAPS_MAP_CENTER_LAT', false)
+  settings['maps']['map_center']['long'] = ENV.fetch('CL_SETTINGS_MAPS_MAP_CENTER_LONG') if ENV.fetch('CL_SETTINGS_MAPS_MAP_CENTER_LONG', false)
+  settings['maps']['zoom_level'] = ENV.fetch('CL_SETTINGS_MAPS_ZOOM_LEVEL').to_i if ENV.fetch('CL_SETTINGS_MAPS_ZOOM_LEVEL', false)
+
+  begin
+    config.save!
+  rescue Exception => e
+    logger = Rails.logger
+    logger.error 'Failed to reload environment variables into the seeded data.'
+    logger.error e.message
+    e.backtrace.each { |line| logger.error line }
+  end
```

Comment:
Should we re-raise the error and fail hard as well? I'm doubting, what's your take?

Reply:
I was thinking to not let issues block the app from running. If someone is stuck with setting the right environment variables or if there would be a bug where the initializer keeps failing, this wouldn't prevent people from being able to use the app.

Reply:
Let me know if you disagree, then I will change it.

Reply:
Ok, is fine